### PR TITLE
Add benchmark tests for encoder apis

### PR DESCRIPTION
**Pull request by aayushsoni111**
Change-Id: I972856d603782d3167f7d002896ce87161292d96

### CODE REVIEW

**Review by harishdm — APPROVED** [review]

File: benchmark/benchmark_test.cpp [hunk 1]
```diff
@@ -23,13 +23,64 @@
...
 
-const std::string kDecTestVectors[] = {
-    "./data/city_night.jpg",    "./data/desert_sunset.jpg", "./data/lamps.jpg",
-    "./data/mountains.jpg",     "./data/desert_wanda.jpg",  "./data/grand_canyon.jpg",
-    "./data/mountain_lake.jpg",
+std::vector<std::string> kDecodeAPITestImages{
+    // 12mp test vectors
+    "mountains.jpg",
+    "mountain_lake.jpg",
+    "desert_wanda.jpg",
```
Comment (DichenZhang1) [inline benchmark/benchmark_test.cpp:30]: Are these images checked in?

File: benchmark/benchmark_test.cpp [hunk 2]
```diff
@@ -46,6 +97,34 @@ std::string ofToString(const ultrahdr_output_format of) {
   }
 }
 
+std::string cgToString(const ultrahdr_color_gamut cg) {
```
Comment (DichenZhang1) [inline benchmark/benchmark_test.cpp:100]: I'd prefer using `colorGamut` because `cg` is too ambiguous.

File: benchmark/benchmark_test.cpp (hunk 1 above)
Comment (harishdm) [reply benchmark/benchmark_test.cpp:30]: these aren't checked in yet. Currently these will need to be manually placed. Once we get a confirmation, we will upload a zip that will contain these files.

**Review by DichenZhang1 — APPROVED** [review]

File: benchmark/benchmark_test.cpp (hunk 1 above)
Comment (DichenZhang1) [reply benchmark/benchmark_test.cpp:30]: These images are shareable.

**Review by harishdm — APPROVED** [review]

**Review by DichenZhang1 — APPROVED** [review]

File: benchmark/benchmark_test.cpp (hunk 1 above)
Comment (DichenZhang1) [reply benchmark/benchmark_test.cpp:32]: Hi Harish, could you confirm if the images are going to be checked in together with this CL?

Comment (harishdm) [reply benchmark/benchmark_test.cpp:32]: @DichenZhang1 Images are uploaded to google cloud and the third commit in this pull request downloads the zip and uses those files.

**Pull request merged**